packages/broker/src/plugins/storage/StoragePlugin.ts

```diff
@@ -78,7 +83,12 @@ export class StoragePlugin extends Plugin<StoragePluginConfig> {
     private async createStorageConfig() {
         const brokerAddress = new Wallet(this.brokerConfig.ethereumPrivateKey).address
         const apiUrl = this.brokerConfig.streamrUrl + '/api/v1'
-        const storageConfig = await StorageConfig.createInstance(brokerAddress, apiUrl, this.pluginConfig.storageConfig.refreshInterval)
+        const storageConfig = await StorageConfig.createInstance(
+            this.pluginConfig.cluster.clusterAddress || brokerAddress, 
+            this.pluginConfig.cluster.clusterSize || 1,
```

Comment:
The schema specifies that `clusterSize` and `myIndexInCluster` are always non-null `numbers` -> no need to have fallback values

Reply:
Yeah it might be better to have one source for default values to avoid any potential confusion.

Reply:
Removed the fallbacks for `clusterSize` and `myIndexInCluster`. Is there a way to encode the logic: 

> if `clusterAddress` is null, use the `brokerAddress`

...in the schema, or is it still necessary to have that logic here?